### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by andremauricio982** [habom2310/Heart-rate-measurement-using-camera#9]
I uncommented the age part in GUI.py

 self.lbl_Age = QLabel(self) #label to show stable HR
 self.lbl_Age.setGeometry(900,120,300,40)
 self.lbl_Age.setFont(font)
 self.lbl_Age.setText("Age: ")

But when I run the code it doesn't work, how was it supposed to work? Will it take a neural network already trained to classify Age?

**Comment by habom2310** [comment]
Age and gender detection use a pre-trained model to predict. It was an old version when I combine age and gender detection to the application, but the problem is that it slowed down the process so I don't use it here any longer.

If you want to have that feature, you can find it in another repository of mine https://github.com/habom2310/People-tracking-with-Age-and-Gender-detection.
